notebooks/book2/29/hmm_poisson_changepoint.ipynb

```diff
@@ -0,0 +1,1013 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    rng_key, rng_normal = jax.random.split(rng_key)
</pre><p>may be you can try changing <code>rng_key</code> to reproduce the results. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/129/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>